action.yml

```diff
@@ -1,11 +1,8 @@
 # Copyright 2020 The MathWorks, Inc.
 
-name: Run MATLAB 
-Tests description: >-
-  Run all tests in a MATLAB project and generate test artifacts. MATLAB includes
-  any files in your project that have a Test label. If your pipeline does not
-  leverage a MATLAB project, then MATLAB includes all tests in the the root of
-  your repository including its subfolders.
+name: Run MATLAB Tests
+description: >-
+  Run all tests in a MATLAB project and generate test artifacts
```

Comment:
How about this?

Run MATLAB and Simulink tests and generate artifacts

Reply:
Im on board with houman's suggestion